src/app/objective/objective.component.ts

```diff
@@ -121,6 +137,6 @@ export class ObjectiveComponent implements OnInit {
   }
 
   enableAssignButton(): boolean {
-    return !!this.isEditingEnabled && this.objective!.resourceEstimate > 0;
+    return this.hasPeopleAvailable() && !!this.isEditingEnabled && this.objective!.resourceEstimate > 0;
```

Comment:
Minor performance consideration: we should probably check `hasPeopleAvailable()` last, since the other checks are extremely cheap and may avoid the need to call it:

```typescript
return !!this.isEditingEnabled && this.objective!.resourceEstimate > 0 && this.hasPeopleAvailable();
```

Reply:
Fixed it, thanks!